client/src/lib/utils/faucetRequest.ts

```diff
@@ -1,57 +1,54 @@
 import { PUBLIC_DEMO_MODE as DEMO } from "$env/static/public";
+
 import type { NetworkData } from "./networkData";
 
 export async function request(
-	address: string,
-	recaptcha: string,
-	network: NetworkData,
-	parachain?: number
+  address: string,
+  recaptcha: string,
+  network: NetworkData,
+  parachain?: number,
 ): Promise<string> {
-	if (DEMO) {
-		return boilerplateRequest(address, recaptcha);
-	}
-	const chain = parachain && parachain > 0 ? parachain.toString() : undefined;
-	return faucetRequest(address, recaptcha, network, chain);
+  if (DEMO !== undefined && DEMO !== "") {
+    return await boilerplateRequest(address);
```

Comment:
Isn't `if (DEMO)` achieving the same result (and also including if DEMO is not null?

Reply:
> Isn't `if (DEMO)` achieving the same result 

Yes, but implicit boolean casts violate our codestyle

> (and also including if DEMO is not null?

how an env variable would end up being null?